src/js/libcs/Operators.js

```diff
@@ -4530,3 +4530,68 @@ evaluator.dropped$0 = function() {
 evaluator.droppoint$0 = function() {
     return dropPoint;
 };
+
+evaluator.parsecsv$1 = function(args, modifs) {
+    // more strict float parsing
+    var filterFloat = function(value) {
+        if (/^(\-|\+)?([0-9]+(\.[0-9]+)?|Infinity)$/
+            .test(value))
+            return Number(value);
+        return NaN;
+    };
```

Comment:
What is the intention here? In particular, is it deliberate to disallow `1e17`, `0.`, `.5` or `NaN`?

Furthermore, is there any reason not to write this conversion the same as the other autoconversions, i.e. as `if ((/…/).test(itm) && aucon) { data[i][j] = … }` in the case distinctions below? Would make things more consistent in my opinion.